ops/helper-scripts/set-up-git-hooks.sh

```diff
@@ -20,7 +19,7 @@ temp_file=$(mktemp)
     head -n 1 ../../.git/hooks/pre-commit
     cat pre-commit-hook-content
     tail -n +3 ../../.git/hooks/pre-commit
-} > "$temp_file"
+} >"$temp_file"
```

Comment:
Was this flagged by the check? It seems more clear to me to separate the operator from the file name.

Reply:
Visual studio code format document removed that space. Undo the change here since its not a lint issue